api/serializers.py

```diff
@@ -55,12 +58,15 @@ class Meta:
 
 
 class DomainAddressSerializer(PremiumValidatorsMixin, serializers.ModelSerializer):
+    type = serializers.CharField(default="custom")
     class Meta:
         model = DomainAddress
         fields = [
+            "type",
```

Comment:
suggestion: If this is read-only, it should go a few lines down, by "id" and "address".